one_fm/fixtures/doctype_layout.json

```diff
@@ -1013,6 +1006,13 @@
     "parentfield": "fields",
     "parenttype": "DocType Layout"
    },
+   {
```

Comment:
KIndly run migrations on your local bench, and confirm that this works as excpected